docs/connection.md

```diff
@@ -101,12 +101,14 @@ To disable TLS verification set it to `false`. **NOT recommended for production*
 ```
     return [
         'connection' => [
-            'host' => 'localhost',
-            'port' => '9200',
-            'scheme' => 'https',
-            'ssl' => [
-                'cert' => ['path/to/cert.pem', 'passphrase'],
-                'key' => ['path/to/key.pem', 'passphrase'],
+            'Hosts' => ['localhost:9200'],
+            'SSLCert' => [
+                'cert' => './path/to/cert.crt',
+                'password' => null,
+            ],
+            'SSLKey' => [
+                'key' => './path/to/key.key',
+                password' => null,
```

Comment:
I really think we should be backwards compatible with the old config, I rather not ask our users to rewrite their configs after an update

Reply:
I'll look into it and adjust, I believe this was also a remnant from the original PR that didn't have passing tests.